stacks/core/addons/connector.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+
+services:
+  connector:
+    image: cesarbr/knot-fog-connector:dev
+    env_file: './env.d/knot-connector.env'
+    volumes:
```

Comment:
Remove this binding and explain in the documentation that the developer must update it.